clang/lib/CodeGen/CoverageMappingGen.cpp

```diff
@@ -1847,30 +1850,18 @@ struct CounterCoverageMappingBuilder
     // Extract the Parent Region Counter.
     Counter ParentCnt = getRegion().getCounter();
 
-    // Extract the MCDC condition IDs (returns 0 if not needed).
-    MCDCConditionID NextOrID = MCDCBuilder.getNextLOrCondID();
-    MCDCConditionID NextAndID = MCDCBuilder.getNextLAndCondID();
     MCDCConditionID LHSid = MCDCBuilder.getCondID(E->getLHS());
     MCDCConditionID RHSid = MCDCBuilder.getCondID(E->getRHS());
```

Comment:
They may be moved to `createBranchRegion()` since `CondID` is available as `CondIDMap`.